apps/ejabberd/test/auth_jwt_SUITE.erl

```diff
@@ -126,28 +149,52 @@ store_type(_C) ->
 dirty_get_registered_users(_C) ->
     [] = ejabberd_auth_jwt:dirty_get_registered_users().
 
+check_password_succeeds_for_pubkey_signed_token(C) ->
+    Key = proplists:get_value(priv_key, C),
+    true = ejabberd_auth_jwt:check_password(?USERNAME, ?DOMAIN1, generate_token(rs256, 0, Key)).
+
 %%--------------------------------------------------------------------
 %% Helpers
 %%--------------------------------------------------------------------
 
+ets_owner(Parent) ->
+    ets:new(jwt_meck, [public, named_table, set]),
+    Parent ! ets_ready,
+    receive stop -> ok end.
+
+wait_for_ets() -> receive ets_ready -> ok end.
```

Comment:
It's a bit tricky to debug suites when some of them stuck in waiting. And we don't use time traps.
So, if something goes wrong, we would stuck in 30 minutes here.
I would suggest to have:

```
after 5000 -> ct:fail(help_me_please)
```